Configuration/Test/test.py

```diff
@@ -0,0 +1,534 @@
+    sourceSeed = cms.PSet(initialSeed =cms.untracked.uint32(98765)),
+    generator = cms.PSet(initialSeed = cms.untracked.uint32(98766)),
+    SmearingGenerator = cms.PSet(initialSeed =cms.untracked.uint32(3849)),
+    T2Digis = cms.PSet(initialSeed =cms.untracked.uint32(98765)),
+    T2MCl = cms.PSet(initialSeed =cms.untracked.uint32(24141)),
+    RPFastStationSimulation = cms.PSet(initialSeed =cms.untracked.uint32(12)),
+    RPFastFullSimulation = cms.PSet(initialSeed =cms.untracked.uint32(13)),
+    mix = cms.PSet(initialSeed = cms.untracked.uint32(24141)),
+    LHCTransport = cms.PSet(initialSeed = cms.untracked.uint32(24143), engineName = cms.untracked.string('TRandom3')
+  )
+
+)
+################# STEP 2 process.SmearingGenerator
+process.SmearingGenerator = cms.EDProducer("GaussEvtVtxEnergyGenerator",
+    src   = cms.InputTag("generator"),
+    MeanX = cms.double(0.0),
+    MeanY = cms.double(0.0),
+    MeanZ = cms.double(0.0),
+    SigmaX = cms.double(0.000001),
+    SigmaY = cms.double(0.000001),
+    SigmaZ = cms.double(0.000001),
+    TimeOffset = cms.double(0.0)
+)
+# declare optics parameters
+#process.load("Configuration.TotemOpticsConfiguration.OpticsConfig_6500GeV_0p8_145urad_cfi")
+process.BeamOpticsParamsESSource = cms.ESSource("BeamOpticsParamsESSource",
+    BeamEnergy = cms.double(6500.0), # Gev
+    ProtonMass = cms.double(0.938272029), # Gev
+    LightSpeed = cms.double(300000000.0),
+    NormalizedEmittanceX = cms.double(3.75e-06),
+    NormalizedEmittanceY = cms.double(3.75e-06),
+    BetaStarX = cms.double(0.4), # m
+    BetaStarY = cms.double(0.4), # m
+    CrossingAngleX = cms.double(185e-6),
+    CrossingAngleY = cms.double(0.0),
+    BeamDisplacementX = cms.double(0.0), # m
+    BeamDisplacementY = cms.double(0.0), # m
+    BeamDisplacementZ = cms.double(0.0), # m
+    BunchSizeZ = cms.double(0.07), # m
+    MeanXi = cms.double(0.0), # energy smearing
+    SigmaXi = cms.double(0.0001)
+)
+
+process.ProtonTransportFunctionsESSource = cms.ESProducer("ProtonTransportFunctionsESSource",
+    opticsFile = cms.string(''), # automatic
+    maySymmetrize = cms.bool(True), # this optic is assymmetric
+    verbosity = cms.untracked.uint32(0)
+)
+
+BeamProtTransportSetup = cms.PSet(
+    Verbosity = cms.bool(False),
+    ModelRootFile = cms.string('Geometry/VeryForwardProtonTransport/data/parametrization_6500GeV_0p4_185_reco_beam1.root'),
+    Model_IP_150_R_Name = cms.string('ip5_to_beg_150_station_lhcb1'),
+    Model_IP_150_L_Name = cms.string('ip5_to_beg_150_station_lhcb1'),
+
+    # in m, should be consistent with geometry xml definitions
+    Model_IP_150_R_Zmin = cms.double(0.0),
+    Model_IP_150_R_Zmax = cms.double(202.769),
+    Model_IP_150_L_Zmax = cms.double(-202.769),
+    Model_IP_150_L_Zmin = cms.double(0.0),
+)
+
+
+# ################## STEP 3 process.g4SimHits
+#
+# # Geometry - beta* specific
+# process.load("Geometry.VeryForwardGeometry.geometryRP_cfi")
```

Comment:
Why is it unused ?

Reply:
In file Geometry.VeryForwardGeometry.geometryRP_cfi geometry is practically the same except some namings like:
XMLIdealGeometryESSource_CTPPS instead of process.XMLIdealGeometryESSource

Reply:
Should I create own Geomatry config or deal with naming?

Reply:
Make it consistent in some way. In principle there should be only one geometry definition.
Does everything work if XMLIdealGeometryESSource_CTPPS is skipped ?
I remember some time ago we had to do some `copy.deepcopy` to make it working.

What about root node name ? Is it the same in both locations ?

Reply:
Root node is same in my config, and geometry imported by reconstruction.
Actually all stuff is the same except this CTTPS suffix in process attribute name.
When process.load("Geometry.VeryForwardGeometry.geometryRP_cfi")
I get 'NoProxyException': No data of type "DDCompactView" with label "" in record "IdealGeometryRecord"